index.js

```diff
@@ -1,20 +1,42 @@
 import {HTML5Video, Log, Events} from 'clappr'
 import shaka from 'shaka-player'
 
-const SEND_STATS_AT = 30 * 1000
-const AUTO = -1
-const SHAKA_READY = 'shaka:ready'
+const SEND_STATS_INTERVAL_MS = 30 * 1e3
+const DEFAULT_LEVEL_AUTO = -1
 
-export default class DashShakaPlayback extends HTML5Video {
-  get name() {return 'dash_shaka_playback'}
+class DashShakaPlayback extends HTML5Video {
+  static get Events () {
+    return {
+      SHAKA_READY: 'shaka:ready'
+    }
+  }
+
+  static canPlay (resource, mimeType = '') {
+    shaka.polyfill.installAll()
+    var browserSupported = shaka.Player.isBrowserSupported()
+    var resourceParts = resource.split('?')[0].match(/.*\.(.*)$/) || []
+    return browserSupported && ((resourceParts[1] === 'mpd') || mimeType.indexOf('application/dash+xml') > -1)
+  }
+
+  get name () {
+    return 'DashShakaPlayback'
```

Comment:
Hi guys
I know this PR has already been closed (a great contribution, btw), but I think that changing the plugin name can cause side effects in projects that already use this plugin and hope that this value will not (never) be changed, so this seems like a breaking change.
Also, the standard of _Clappr's_ plugins names is `snake_case`, isn't?
What do you think @leandromoreira @tchakabam?